frontend/src/employee-frontend/components/unit/TabCalendar.tsx

```diff
@@ -279,7 +286,16 @@ export default React.memo(function TabCalendar() {
           )}
 
         <StickyTopBar>
-          <div data-qa-week-range={week.dateRange.toString()} />
+          <div
+            data-qa-date-range={
+              mode === 'week'
+                ? week.dateRange.toString()
+                : new FiniteDateRange(
+                    selectedDate.startOfMonth(),
+                    selectedDate.lastDayOfMonth()
+                  ).toString()
+            }
+          />
```

Comment:
I think this could be moved inside `ActiveDateRangeSelector` component so that you don't have to form the actual date range twice.

Reply:
Good idea, done.